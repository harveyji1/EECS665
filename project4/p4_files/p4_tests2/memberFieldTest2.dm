Person : class {
    age : int;
    
};

main : () int {
    a : Person;
    b : int = a--age;
    return a--age;
}