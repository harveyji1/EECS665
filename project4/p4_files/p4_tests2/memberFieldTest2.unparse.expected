Person{Person} : class {
    age{int} : int;
};
main{()->int} : () int  {
    a{Person} : Person{Person};
    b{int} : int = a{Person}--age{int};
    return a{Person}--age{int};
}